src/containers/Experiment/SamplesTable.js

```diff
@@ -370,7 +361,17 @@ function ThComponent({ toggleSort, className, children, ...rest }) {
 }
 
 function AddRemoveCell({ original: sample, row: { id: rowId } }) {
-  const { experimentAccessionCode } = sample;
+  // retrieve all experiment accession codes referencing this sample
+  const { experimentAccessionCodes } = sample;
+  // Create a dataset slice, where we include all experiments that are referencing this sample
+  // that way when it get's added/removed it will impact all those experiments
```

Comment:
Typo: `get's` should be `gets`. You may also want to add a comma or semicolon at the end of the previous line.